_summaries/kang2019decoupling_2.md

```diff
@@ -1,19 +1,86 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: kang2019decoupling
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: joshpapermaster
+score: 7 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
+<!-- TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+## What is the core idea?
+Visual recognition solutions can struggle when there is a long-tailed distribution, which is also known as the class imbalance problem. This paper separates the this problem into two processes: representation learning and classification. 
+
+Representation learning
+- trained with different sampling techniques and losses
+
+Classification
+- the long tailed classes are recognized by the model from various classifiers  
+
+The paper determined the following findings from using their decoupled process:
+- instance based sampling provided really good generalized results
+- during representation learning it's beneficial to adjust decision boundaries of classifier 
```

Comment:
I don't think this is what they meant. They meant it's beneficial to adjust decision boundaries of classifier during the phase of classifier training. But the decision boundaries were originally learned during representation learning, which is the first stage of the process.